### PR TITLE
[14.0][MIG] shopinvader_delivery_instruction from 13.0

### PR DESCRIPTION
**Pull request by acsonefho**
* [x] Depends on https://github.com/OCA/sale-workflow/pull/1769

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sale-workflow https://github.com/acsone/sale-workflow 14.0-mig-sale_stock_picking_note
```
Comment (rousseldenis) [inline oca_dependencies.txt]: @acsonefho In shopinvader, use test-requirements.txt instead.

**Comment by lmignon** [comment]
@acsonefho Can you finalize this one? Thank you!

**Comment by acsonefho** [comment]
@shopinvader/shopinvader-maintainers

**Review by PierrickBrun — APPROVED** [review]
LGTM

**Pull request merged**